test/y2network/config_reader/sysconfig_test.rb

```diff
@@ -109,4 +109,16 @@
       end
     end
   end
+
+  describe "#forward_ipv4?" do
+    it "returns true when IPv4 forwarding is allowed" do
+      expect(reader.send(:forward_ipv4?)).to be_truthy
```

Comment:
Why truthy? Please, if you expect `true`, then check for `true` (use `eq true` or `be true`). Otherwise, any value which is different to `false` and `nil` will be OK.